src/lib.rs

```diff
@@ -15,8 +15,8 @@ pub use helpers::*;
 pub mod bins;
 pub mod fetchers;
 
-/// Compiled target triple, used as default for binary fetching
-pub const TARGET: &str = env!("TARGET");
+mod target;
+pub use target::{detect_targets, TARGET};
```

Comment:
Nitpick : the style so far is more to use a star re-export, reckon we should just stick with it for now

Reply:
Done!